bundles/org.eclipse.emfcloud.modelserver.common/src/org/eclipse/emfcloud/modelserver/common/patch/JsonPatchTestException.java

```diff
@@ -0,0 +1,25 @@
+/********************************************************************************
+ * Copyright (c) 2022 STMicroelectronics.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.emfcloud.modelserver.common.patch;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+/**
+ * An exception thrown when a Json Patch "test" operation fails.
+ */
+public class JsonPatchTestException extends Exception {
+
+   private static final long serialVersionUID = 8333555068740826391L;
+
+   public JsonPatchTestException(final JsonNode testOperation) {
```

Comment:
This should be implemented properly if/when we support test operations. If the Exception is meant to give detailed error informations, it needs to know what was being tested to know what failed. But since it's not really implemented, the parameter is currently unused. I don't know if the detailed error message will come from the exception itself, or from the class that creates the exception. In one case, it makes sense to have the test exception; in the other case it can be replaced with a simple preformatted string parameter.

Reply:
Understood, let's leave it for now.